.github/workflows/release.yml

```diff
@@ -344,21 +344,22 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           builder: ${{ steps.buildx-builder.outputs.name }}
-          push: false
+          push: true
           context: .
           file: Dockerfile.localstatic
           platforms: linux/amd64 #,linux/arm64
           tags: |
-            localstatic
+            wasmcloud.azurecr.io/wasmcloud_static_base:${{ env.app-version }}
       - name: Build and release static docker image
         uses: docker/build-push-action@v3
         with:
           builder: ${{ steps.buildx-builder.outputs.name }}
           push: true
           context: .
           file: ${{ env.app-name }}/Dockerfile.static
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64 #,linux/arm64
```

Comment:
Should this be commented in? Ignore if just testing.

Reply:
no, we're having issues with the cross-compiled builds